cmd/krew/main.go

```diff
@@ -22,5 +22,5 @@ import (
 
 func main() {
 	cmd.Execute()
-	defer klog.Flush()
+	klog.Flush()
```

Comment:
🤔 let's move this 1 above maybe and keep as defer.

there might be some os.Exit/panics causing klog.Flush to not be called

Reply:
Sounds good. For some reason the linter started complaining about the defer being at the end of the function
```
$ hack/run-lint.sh
cmd/krew/main.go:25:2: unnecessaryDefer: defer klog.Flush() is placed just before return (gocritic)
        defer klog.Flush()
```

I'll do as you suggested :+1: